index.bs

```diff
@@ -466,15 +521,13 @@ spec: webrtc-pc; urlPrefix: https://www.w3.org/TR/webrtc/#
           return a Promise rejected with a <code>TypeError</code>.
         </li>
         <li>
-          If <code>configuration.video</code> in non-null and
-          <code>configuration.video.contentType</code> is not a <a>valid
-          VideoConfiguration contentType</a>, return a Promise rejected with a
+          If <code>configuration.video</code> is <a>present</a> and is not a
```

Comment:
I'm fine with the algo text you linked for rejecting the unsupported (or just whacky) types. I think throwing a type error for malformed strings is still the right way to go to save users a lot of head scratching over typos. With the algo you linked and the current type error text, we're basically good to go except that the "validity" definition needs to be updated to not *always* require codecs= (mentioned in my other comments). With that addressed I think we're good to land